qemu/tests/qemu_guest_agent.py

```diff
@@ -354,14 +351,14 @@ def gagent_check_reboot(self, test, params, env):
         error_context.context("Verify serial output has '%s'" % pattern)
         rebooted = self.__gagent_check_serial_output(pattern)
         if not rebooted:
-            raise error.TestFail("Could not reboot VM via guest agent")
+            raise test.fail("Could not reboot VM via guest agent")
```

Comment:
~~raise~~

also applies for the ones below.

Reply:
update